Sources/MapboxNavigation/SpeedLimitStyleKit.swift

```diff
@@ -91,6 +91,82 @@ public class SpeedLimitStyleKit : NSObject {
 
     }
 
+    @objc dynamic public class func drawMUTCDForUS(frame targetFrame: CGRect = CGRect(x: 0, y: 0, width: 68, height: 85), resizing: ResizingBehavior = .aspectFit, signBackColor: UIColor = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1.000), strokeColor: UIColor = UIColor(red: 0.000, green: 0.000, blue: 0.000, alpha: 1.000), limit: String = "50", legend: String = "SPEED LIMIT") {
```

Comment:
It seems to have issue with the frame size in `CarPlay` in active navigation. It would lead to the loss of the `CompassView` and the `SpeedLimitView`. And I think we should make the width of the `SpeedLimitView` as same as the `CompassView` in visualization on CarPlay with the new design.